doc/migrations/v0.28-v0.29.md

```diff
@@ -90,6 +90,36 @@ libp2p.pubsub.on(topic, handler)
 libp2p.pubsub.subscribe(topic)
 ```
 
+In the latest releases, despite not being documented in `libp2p` the underlying pubsub routers supported subscribing to multiple topics at the same time. We removed that code complexity, since this is easily achieved in the application layer if needed.
```

Comment:
```suggestion
In the latest release, despite not being documented in `libp2p` the underlying pubsub routers supported subscribing to multiple topics at the same time. We removed that code complexity, since this is easily achieved in the application layer if needed.
```